Library/Homebrew/brew.sh

```diff
@@ -353,6 +353,16 @@ then
   export HOMEBREW_FORCE_HOMEBREW_ON_LINUX="1"
 fi
 
+HOMEBREW_CORE_REPOSITORY_ORIGIN="$(git -C "${HOMEBREW_CORE_REPOSITORY}" remote get-url origin)"
```

Comment:
I think we should move this whole section to the place after all `git` checks below.

Reply:
And then also use HOMEBREW_GIT instead of plain git command.

Reply:
Done